theseus_gui/src/pages/project/Version.vue

```diff
@@ -0,0 +1,350 @@
+          <div class="metadata-item">
+            <span class="metadata-label">Loaders</span>
+            <span class="metadata-value">{{
+              version.loaders
+                .map((loader) => loader.charAt(0).toUpperCase() + loader.slice(1))
+                .join(', ')
+            }}</span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Game Versions</span>
+            <span class="metadata-value"> {{ version.game_versions.join(', ') }} </span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Downloads</span>
+            <span class="metadata-value">{{ version.downloads }}</span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Publication Date</span>
+            <span class="metadata-value">
+              {{
+                new Date(version.date_published).toLocaleString('en-US', {
+                  month: 'long',
+                  day: 'numeric',
+                  year: 'numeric',
+                })
+              }}
+              at
+              {{
+                new Date(version.date_published).toLocaleString('en-US', {
+                  hour: 'numeric',
+                  minute: 'numeric',
+                  second: 'numeric',
+                  hour12: true,
+                })
+              }}
+            </span>
+          </div>
+          <div v-if="author" class="metadata-item">
+            <span class="metadata-label">Author</span>
+            <a
+              :href="`https://modrinth.com/user/${author.user.username}`"
+              rel="external"
+              target="_blank"
+              class="metadata-value btn author"
+            >
+              <Avatar size="sm" :src="author.user.avatar_url" circle />
+              <span>
+                <strong>
+                  {{ author.user.username }}
+                </strong>
+                <br />
+                {{ author.role }}
+              </span>
+            </a>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Version ID</span>
+            <span class="metadata-value">{{ version.downloads }}</span>
+          </div>
+        </div>
+      </Card>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import {
+  Card,
+  Button,
+  DownloadIcon,
+  FileIcon,
+  Avatar,
+  LeftArrowIcon,
+  ReportIcon,
+  Badge,
+  ExternalIcon,
+  formatBytes,
+  renderString,
+} from 'omorphia'
+import { releaseColor } from '@/helpers/utils'
+import { ofetch } from 'ofetch'
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+
+const [version, members, dependencies] = await Promise.all([
+  ofetch(`https://api.modrinth.com/v2/version/${route.params.version}`),
+  ofetch(`https://api.modrinth.com/v2/project/${route.params.id}/members`),
+  ofetch(`https://api.modrinth.com/v2/project/${route.params.id}/dependencies`),
+])
```

Comment:
Why not just put the members and dependencies in the main func? (since it's async it won't affect speed). You can probs remove the individual version call and just use the find function on the `versions.find((x) => x.id === route.params.version)`